homeworks/app/src/pages/CoursePage.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Page from '../components/common/Page.jsx';
+import CourseLogo from '../components/feature/CourseLogo.jsx';
+import Course from '../components/feature/Course.jsx';
+
+class LoginPage extends React.Component {
+  render() {
+    return (
+      <Page headertext="Course view">
+        <CourseLogo/>
+        <Course/>
+      </Page>
+    );
+  }
+}
+
+export default LoginPage;
```

Comment:
do not use `export default`. It cause bugs like this. You have two components with same name `LoginPage` but you can miss that because  `export default`  allows you to specify any name in import of this file like you use it [here](https://github.com/react-course-epam-05-09-18/ebs-react-course-Igor-Gorchakov/pull/1/files#diff-5384bc18135d0351b332f424691ebf31R5)